src/main/java/com/myaudiolibrary/web/security/SimpleCorsFilter.java

```diff
@@ -0,0 +1,50 @@
+package com.ipiecoles.java.mdd050.security;
```

Comment:
Oublie pas de changer le package pour que cela fonctionne !
```suggestion
package com.myaudiolibrary.web.security;
```